dev-lang/ghc/files/ghc-9.0.2-disable-unboxed-arrays.patch

```diff
@@ -0,0 +1,36 @@
+From f5a511f1ff2cf16e710342948792cb8d7576e27c Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the commit at the top.

Reply:
Done